modules/dr-restoring-cluster-state.adoc

```diff
@@ -553,6 +563,19 @@ $ oc patch etcd cluster -p='{"spec": {"forceRedeploymentReason": "recovery-'"$(
 +
 When the etcd cluster Operator performs a redeployment, the existing nodes are started with new pods similar to the initial bootstrap scale up.
 
+. Turn the quorum guard back on by entering the following command:
++
+[source,terminal]
+----
+$ oc patch etcd/cluster --type=merge -p '\{"spec": {"unsupportedConfigOverrides": null}}
+----
+. You can verify that the `unsupportedConfigOverrides` section is removed from the object by entering this command:
```

Comment:
Nitpick: we could use a newline before this step in the procedure for consistency. :slightly_smiling_face:

Reply:
Good catch, @kowen-rh! 

@lance5890 Can you add the line that Kevin mentioned? After you do that, I can move this along to be merged.